app/templates/teachers/teacher-student-view.jade

```diff
@@ -0,0 +1,220 @@
+extends /templates/base-flat
+
+block page_nav
+  include ../courses/teacher-dashboard-nav.jade
+
+block content
+
+  if !view.user
+    div Loading
+  else
+    .container
+      +breadcrumbs
+      if !me.isAnonymous() && me.isTeacher()
+      - var isOwner = view.classroom ? view.classroom.get('ownerID') === me.id : false;
+        // - var validStudent = ...  
+        // load all students in this class
+        // check if this student is in the class
+          
+      if isOwner
+        // also check if validStudent 
+        h3(class="m-t-2")
+          span(data-i18n="teacher.student_profile")
+          span.spr :
+          span= view.user.get('name')
+        // this user object doesn't have an email property
+        h4(class="student-details" data-i18n="teacher.student_overview")
+        if (view.user.get('email'))
+          div(class="small-details")
+            span(data-i18n="teacher.student_email")
+            span.spr :
+            span= view.user.get('email')
+        else
+          div(class="small-details") 
+            i(data-i18n="teacher.no_email")
+        div(class="small-details")
+          span(data-i18n="user.last_played")
+          span.spr :
+          span= view.lastPlayedString
+          
+        div(class="small-details") 
+          - var status = view.user.prepaidStatus()
+          span(data-i18n='view.user.status')
+          span.spr
+          span(data-i18n="teacher.license_status")
+          span.spr : 
+          strong(class= status === 'expired' ? 'text-danger' : '')= view.studentStatusString()
+        //- 
+        //- svg#visualisation(width="1000", height="520")
+        //- 
+        
+        div.playtime-charts
+          // i18n needed
```

Comment:
Does this still need doing? (Looks like not)

I think I'm the only one to do this, but I often use `(data-i18n="TODO")` so it's really easy to find the ones I haven't done yet. I have a snippet for it, even.